src/App.tsx

```diff
@@ -15,48 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const IS_SORTED_ALPHABET = 'A';
+const IS_SORTED_LENGTH = 'L';
+
+function getPreparedGoods(
+  goods: string[],
+  isSorted: string | '',
+  isReversed: boolean,
```

Comment:
Also, you can add it to the interface, like the possibility ( it is not necessary )